forge/forge/op/eval/forge/eltwise_nary.py

```diff
@@ -246,11 +246,12 @@ def decompose(type, attr, dc, inputs):
         new_inputs = []
         for inp in inputs:
             inp_shape = inp.shape.as_list()
-            inp_shape.insert(axis, 1)
-            new_inp = dc.op("reshape", [inp], (*inp_shape,))
+            insertion_index = axis if axis >= 0 else len(inp_shape) + axis + 1
```

Comment:
Why do we need to convert to positive indexing?

Reply:
With negative axis, insertion is not applied correctly
When axis = -4, output shape is (2, 1, 256, 24, 24) while expected shape from eval function is (1, 2, 256, 24, 24), this is being rectified when axis is being converted to negative